x-pack/plugin/src/test/resources/rest-api-spec/api/ccr.forget_follower.json

```
@@ -1,7 +1,7 @@
 {
   "ccr.forget_follower":{
     "documentation":{
-      "url":"http://www.elastic.co/guide/en/elasticsearch/reference/current"
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current"
```

Not related to your changes, but this link is wrong.
```suggestion
      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/ccr-post-forget-follower.html"
```